src/Helper.ts

```diff
@@ -84,6 +84,12 @@ export class Helper {
           ServiceProfile.SEARCH_1_AUTO_COMPLETE
         );
       }
+
+      if (!autoCompleteService) {
+        autoCompleteService = service.getService(
+          ServiceProfile.SEARCH_2_AUTO_COMPLETE
+        );
+      }
```

Comment:
This is fine for now, but it makes me wonder if it would be better to have a `getFirstMatchingService` method that takes an array of profiles, and returns the first matching result -- then we could replace these long series of if checks with a single lookup.